### PR TITLE
Issue #72: Created a resilience4j-metrics module for Dropwizard metri…

### PR DESCRIPTION
**Pull request by RobWin**
…cs and moved the Stopwatch class into a new resilience4j-core module.

### CODE REVIEW

File: resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/Metrics.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    /**
+     * Creates a timed checked supplier.
+
+     * @param timer the timer to use
+     * @param supplier the original supplier
+     * @return a timed supplier
+     */
+    static <T> Try.CheckedSupplier<T> decorateCheckedSupplier(Timer timer, Try.CheckedSupplier<T> supplier){
+        return () -> {
+            Timer.Context context = timer.time();
```
Comment (storozhukBM) [inline resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/Metrics.java]: You can use try-with-resources
```java
try (Timer.Context context = timer.time()) {
      return supplier.get();
}
```

**Comment by RobWin** [comment]
Thx. Review findings should be implemented.

**Comment by storozhukBM** [comment]
Build fix required though

**Comment by RobWin** [comment]
@storozhukBM Can you pull the branch and compile it locally? I can compile it locally. Not sure why MetricsTest does not find the package `io.github.resilience4j.test`.

**Comment by RobWin** [comment]
Thx.

**Pull request merged**